Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -199,6 +217,8 @@ namespace java::net
         HttpURLConnection(jobject object);
 
         int GetResponseCode() const;
+
+        void SetRequestMethod(std::string requestMethod);
```

Comment:
To avoid copying, this is probably better:
```suggestion
        void SetRequestMethod(const std::string& requestMethod);
```